pytg/fix_msg_array.py

```diff
@@ -41,12 +42,19 @@ def fix_message(message):
 
 
 def fix_peer(peer):
+	# rename peer_type => type
+	if peer["peer_type"]:
```

Comment:
To check for existence use `if "peer_type" in peer`.
That above only checks the value of that attribute.
It will fail, if there is no `"peer_type"` key.